### PR TITLE
move cli docs

### PR DESCRIPTION
**Pull request by denis-yuen**
**Description**
Move CLI docs to CLI repo

**Review Instructions**
n/a

**Issue**
https://github.com/dockstore/dockstore/pull/5979

**Security**
If there are any concerns that require extra attention from the security team, highlight them here.

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Check that you pass the basic style checks and unit tests by running `./mvnw clean install` 
- [x] If this PR is for a user-facing feature, create and link a documentation ticket for this feature (usually in the same milestone as the linked issue). Style points if you create a documentation PR directly and link that instead.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ The following section is useful for users of Dockstore (e.g. those that want to
 launch tools). 
 
 After registering at [dockstore.org](https://dockstore.org), you will be able to download the Dockstore 
-CLI at https://dockstore.org/onboarding
+CLI at https://dockstore.org/onboarding The dockstore command line script should be installed in a location in your path.
```
Comment (kathy-t) [inline README.md:33]: Missing period between `https://dockstore.org/onboarding` and `The dockstore command line script should be installed in a location in your path`

Comment (kathy-t) [inline README.md:33]: dockstore -> Dockstore

**Review by kathy-t — APPROVED** [review]

**Comment by david4096** [comment]
Basically fine but the sentence will be a little awkward "you will be able to download the Dockstore CLI at [https://dockstore.org/onboarding](onboarding)". Would suggest the link have a more descriptive title

**Review by david4096 — APPROVED** [review]

**Comment by denis-yuen** [comment]
> Basically fine

actually, it was straight-up flipped, fixing

**Pull request merged**